### PR TITLE
Create basic front end

### PR DESCRIPTION
**Pull request by sayan-gg**
This PR creates a basic front-end for GoraPass. It is able to send GET and POST requests to the backend server in order to get the stamps and filtered hike data.

### CODE REVIEW

File: node_modules/.bin/loose-envify
Comment (brandiseeley) [inline node_modules/.bin/loose-envify:1]: We should include all `node_modules` files in the `.gitignore`. Installing the node modules should be part of the setup instructions.

**Review by brandiseeley — CHANGES_REQUESTED** [review]
Not super familiar with React yet, but looks good. The only issue is that this includes all of the `node_modules`. Those should be part of the `.gitignore`

File: node_modules/.bin/loose-envify
Comment (sayan-gg) [reply node_modules/.bin/loose-envify:1]: Fixed this!

**Comment by sayan-gg** [comment]
@brandiseeley ok, managed to fix that issue and the POST requests are working, so I think this is good to merge now.

**Pull request merged**